src/features/vault/redux/fetchBalances.js

```diff
@@ -54,8 +54,11 @@ export function fetchBalances({ address, web3, tokens }) {
 
           balanceResults.forEach(balanceResult => {
             newTokens[balanceResult.symbol] = {
-              ...tokens[balanceResult.symbol],
```

Comment:
That's not a typo, `tokens[balanceResult.symbol]` suppose to preserve existing data from `tokens`